README.md

```diff
@@ -34,19 +42,29 @@ collections:
 - name: community.general
 ```
 
+Note that if you install the collection manually, it will not be upgraded automatically with upgrade of the Ansible package. To upgrade the collection to the latest available version, run the following command:
```

Comment:
I'm not sure about this.
```suggestion
Note that if you install the collection manually, it will not be upgraded automatically with upgrading of the Ansible package. To upgrade the collection to the latest available version, run the following command:
```

Reply:
Maybe this will make it clearer: `it will not be upgraded automatically when you upgrade the Ansible package.`

Reply:
I was about to suggest something similar, but it had more passive voice, so this one is definitely better :)

Reply:
Committed, thanks!